public/app/dashboard/profileController.js

```diff
@@ -125,6 +125,22 @@
     };
 
     /**
+     * Toggle activity breakdown selection (contributions or impacts).
+     *
+     * @param  {String}         filter          Which portion of the breakdown to display
+     * @return {void}
+     */
+    var toggleBreakdown = $scope.toggleBreakdown = function(filter) {
+      $scope.breakdown.selected = filter;
+
+      // Track toggle.
+      analyticsService.track('Toggle breakdown filter', {
```

Comment:
@paulkerschen Is this event name intentionally vague? Is the context clear in MixPanel based on key names below?